### ISSUE TITLE
Multiple accounts

### ISSUE THREAD

**Issue by eddsalkield** [corecode/dma#113]
I'm looking to use `dma` to send mail to different mail servers. Ideally it would choose the server based on the sender address header. I've done some reading but can't find a way to do this. Is it currently possible?

**Comment by corecode** [comment]
you mean multiple "smarthosts"?  That is not possible and out of the 
scope for dma.

**Comment by eddsalkield** [comment]
Fair enough - would you mind explaining why? Having multiple mail accounts is a fairly common use case, I would imagine.

Is there a way in which one could do this with multiple `dma.conf` files? I can't seem to find a way to point `dma` to a particular config file.

**Comment by corecode** [comment]
dma is a simple MTA intended to get locally generated mails off the 
machine.  It's not intended to be a MUA that deals with multiple 
accounts and multiple relaying servers.

What's your use case?

**Comment by ian-abbott** [comment]
Does your client have an offline mode (like Thunderbird)?

**Comment by eddsalkield** [comment]
No - I'm using [aerc](https://aerc-mail.org/) which doesn't have an offline mode. I'm trying to use a tool like dma to buffer mail to send when back online. Any ideas?

**Comment by ian-abbott** [comment]
You could probably set aerc's `outgoing` setting for the account to point to a per-account wrapper script that enqueues the email in a per-account directory to be sent later by [msmtp](https://marlam.de/msmtp/). Have a look at [Using msmtp offline](https://wiki.archlinux.org/title/msmtp#Using_msmtp_offline) and [Using Mutt offline](https://whatsecurity.nl/using-mutt_offline.html) for possible clues how to do this.
